server/services/prometheus/intrumentor.go

```diff
@@ -0,0 +1,27 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// Instrumentor used to instrumentate metrics in prometheus
+type Instrumentor struct {
+	Version string
+}
+
+// NewInstrumentor Factory method to create a new instrumentator
+func NewInstrumentor(version string) *Instrumentor {
+	return &Instrumentor{
+		Version: version,
+	}
+}
+
+// ExposeBuildInfo exposes a gauge in prometheus for build info
+func (i *Instrumentor) ExposeBuildInfo() {
+	promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "focalboard_build_nfo",
+		Help: "Build information of Focalboard",
+	}, []string{"Version"},
+	).WithLabelValues(i.Version).Set(1)
```

Comment:
I'm not sure if I entirely understand why we're setting the gauge to `1`, but should be use `.Inc()` instead to increment the gauge?

Reply:
actually we are not increasing right? we just pass the latest version

Reply:
Correct me if I'm wrong here, but you're creating a gauge for how many Focalboard instances are running a specific version, right? I might be entirely wrong here though.

I'm just not sure why the gauge value is being set to 1.

Reply:
I see your confusion.

> but you're creating a gauge for how many Focalboard instances are running a specific version, right?

No, I am just exposing the version so we can show for example in grafana that these metrics are about X version.

Reply:
@spirosoik got it.